code/src/components/Intro.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Intro = () => {
+    
+    const onButtonClick = () => {
+        window.location.href ="#main-form"
+    }
+
+    return (
+
+        <div className="intro" id="intro">
+            <p>Veggies es bonus vobis, proinde vos postulo essum magis kohlrabi welsh onion daikon amaranth tatsoi tomatillo melon azuki bean garlic.
+            Gumbo beet greens corn soko endive gumbo gourd.</p>
+            <button className="button" onClick={onButtonClick}>Begin here</button>
```

Comment:
You can make it easier :

```
<a href="#main-form">Begin here</a>
```

The result will be the same - `href` with hashtag will redirect us always to the same page we are currently in and to that exact `id` element :)